test/integration/consumer_group_test.exs

```diff
@@ -24,6 +24,26 @@ defmodule KafkaEx.ConsumerGroup.Test do
     assert consumer_group == :no_consumer_group
   end
 
+  test "create_worker allows us to pass in use_ssl and ssl_options options" do
+    Application.put_env(:kafka_ex, :use_ssl, true)
```

Comment:
You'd need to reset this to the previous value at the end of your test.

Reply:
Sure, will do